examples/stable_diffusion_xl/configs/inference/sd_xl_refiner.yaml

```diff
@@ -71,7 +69,7 @@ model:
               outdim: 256  # multiplied by one
 
     first_stage_config:
-      target: gm.models.autoencoder.AutoencoderKLInferenceWrapper
+      target: gm.models.autoencoder.AutoencoderKL
```

Comment:
Please maintain the original configuration

refer to
ori official code: [45c443b316737a4ab6e40413d7794a7f5657c19f](https://github.com/Stability-AI/generative-models/tree/45c443b316737a4ab6e40413d7794a7f5657c19f)
new official code: https://github.com/Stability-AI/generative-models/blob/main/sgm/modules/autoencoding/regularizers/__init__.py#L24

Reply:
Reverted